src/test/java/org/cactoos/io/TeeInputFromBytesTest.java

```diff
@@ -57,11 +58,15 @@ public void copiesFromBytesToPath() throws IOException {
         final File output = this.folder.newFile();
         new Assertion<>(
             "Must copy bytes to file path",
-            new TeeInput(
-                new BytesOf(message),
-                output.toPath()
+            new TextOf(
+                new TeeInput(
+                    new BytesOf(message),
+                    output.toPath()
+                )
             ),
-            new InputHasContent(message)
+            new TextIs(
+                new TextOf(output.toPath())
+            )
```

Comment:
@marceloamadeu this doesn't seem to exactly cover the todo intention: instead of coparing the content of the teeinput, you comparend the content of the teeinput with the content of the file the teeinput ouputs to.
This is actually worst than before, because you don't even test the message is the right one.

I believe the objective here is to compare the content of the file with the message, that's all.

Reply:
@victornoel Please check that it is now correct. I have used examples that are in [Input/Output](https://github.com/yegor256/cactoos#inputoutput)

Reply:
@marceloamadeu this is better, please just use new `InputHasContent` to validate the content of the file instead of going through an extra layer via `TextOf`. We try to depends on the minimum of other classes in the tests to make them more robust.

Reply:
@victornoel Changed ... please check.